src/SFA.DAS.EmployerAccounts.Web/Mappings/AgreementMappings.cs

```diff
@@ -47,9 +41,6 @@ public AgreementMappings()
                 .ForMember(dest => dest.SignedDateText, opt => opt.Ignore())
                 .ForMember(dest => dest.AccountLegalEntityPublicHashedId, opts => opts.Ignore());
 
-            CreateMap<AgreementTemplateDto, AgreementTemplateViewModel>()
```

Comment:
Still require this line:
`CreateMap<AgreementTemplateDto, AgreementTemplateViewModel>();`